### DISCUSSION TITLE
Does the "fediverse:creator" tag also apply to sub-domains or would one need to also include subdomains?

### DISCUSSION THREAD

**Discussion by Andre601** [mastodon/mastodon#32929] [Q&A] [answered]
I was wondering, if subdomains would also work when only adding the main domain itself.

As an example, if I add `example.com` to the domain list in the Author Attribution section, would it also work for something like `blog.example.com` if the site itself has the meta tag? Or would I need to add every sub-domain separately?

If it requires sub-domains to be included in the list, then this should be mentioned clearly within the settings, or else people might get confused.

**Comment by Andre601** [comment]
I want to make something clear, which is that from my perspective as a basic Mastodon user, the current docs on the settings page are... bare bones. And not in a good way.

While it may explain the basic concept of setting things up, does it leave open the question on things like I just asked (Wether subdomains are included for a domain or need to be added separately).

Perhaps it would be good to have a dedicated page in the mastodon docs and link to them in the Settings window, to allow a much more detailed explanation of this feature, like recommended places to include the tag and maybe even some examples for specific page-builders (jekyll, Svelte, 11ty, MkDocs, etc), so people have an easier time understanding this feature.

But either way, current docs of the feature lack some clearance.

**Comment by trwnh** [comment, marked as answer]
From a little digging, it looks like it does include wildcard subdomains: https://github.com/mastodon/mastodon/pull/31819#issuecomment-2342457488

I'll create some issues here to note this on the settings page, and a docs issue to note this in the user guide

**Reply by trwnh** [reply]
- https://github.com/mastodon/mastodon/issues/32931
